arangod/Aql/AqlItemBlockInputMatrix.cpp

```diff
@@ -163,6 +155,21 @@ size_t AqlItemBlockInputMatrix::skipAllRemainingDataRows() {
     }
     resetBlockIndex();
   }
+}
+
+size_t AqlItemBlockInputMatrix::skipAllRemainingDataRows() {
```

Comment:
looks like this can be made `noexcept` as well
```suggestion
size_t AqlItemBlockInputMatrix::skipAllRemainingDataRows() noexcept {
```